test/unittests/execute_control_test.cpp

```diff
@@ -28,33 +35,23 @@ TEST(execute_control, nop)
 
 TEST(execute_control, block_br)
 {
-    // (local i32 i32)
-    // block
-    //   i32.const 0xa
-    //   local.set 1
-    //   br 0
-    //   i32.const 0xb
-    //   local.set 1
-    // end
-    // local.get 1
-
-    Module module;
-    module.codesec.emplace_back(Code{2,
-        {Instr::block, Instr::i32_const, Instr::local_set, Instr::br, Instr::i32_const,
-            Instr::local_set, Instr::end, Instr::local_get, Instr::end},
-        from_hex("00"       /* arity */
-                 "07000000" /* target_pc: 7 */
-                 "1d000000" /* target_imm: 29 */
-                 "0a000000" /* i32.const 0xa */
-                 "01000000" /* local.set 1 */
-                 "00000000" /* br 0 */
-                 "0b000000" /* i32.const 0xb */
-                 "01000000" /* local.set 1 */
-                 "01000000" /* local.get 1 */
-            )});
-
-    const auto [trap, ret] = execute(module, 0, {});
+    /* wat2wasm --no-check
```

Comment:
This code doesn't need `--no-check` though, right?

Reply:
It does because function result type is `[]` but the instructions in the functions give `[i32]`